### PR TITLE
[modified] CTF mapcycle with #mapcycle results

### PR DESCRIPTION
**Pull request by Mazey**
## Status

- **READY**

## Description

March 2019 poll 1: Add 1 new map: ffsff_PlatformFlats, re-add 1 map from the returning maps pool. Moved Punk123_PrincessCity2 to the returning maps pool.

### CODE REVIEW

File: Rules/CTF/mapcycle.cfg [hunk 1]
```diff
@@ -16,11 +16,14 @@ mapcycle =
...
 				#Ej
 			Maps/CTF/Ej_Cataracts.png;
+			Maps/CTF/Ej_Chambers.png;
 				#Ferrezinhre
 			Maps/CTF/Ferrezinhre_Highlands.png;
 				#Geti
 			Maps/CTF/HearthPlains.png;
 			Maps/CTF/Snatchmark.png;
+				#ffsff
+			Maps/CTF/PlatformFlats.png;
```
Comment (furai) [inline Rules/CTF/mapcycle.cfg]: Wrong file name.

Comment (Mazey) [reply Rules/CTF/mapcycle.cfg]: Please merge and fix? You know I don’t have PC access.

Comment (furai) [inline Rules/CTF/mapcycle.cfg]:
```suggestion
			Maps/CTF/ffsff_PlatformFlats.png;
```

See if you can accept that suggestion from phone. If not I'll do it manually after merging.

Comment (Mazey) [reply Rules/CTF/mapcycle.cfg]: Accepted, but I don’t see a commit anywhere, not sure.

**Pull request merged**